code/game/objects/items/apc_frame.dm

```diff
@@ -6,6 +6,7 @@
 	icon = 'icons/obj/apc_repair.dmi'
 	icon_state = "apc_frame"
 	refund_amt = 2
+	matter = list(DEFAULT_WALL_MATERIAL = 100,"glass" = 30)

```

Comment:
Please use MAT_STEEL and MAT_GLASS for these and the other lines - DEFAULT_WALL_MATERIAL being used for matter lists was really a mistake in the first place and using bare strings for glass is not ideal.